### PR TITLE
Update cs.lua

### PR DESCRIPTION
**Pull request by stepanvalic**
## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Review by solareon — APPROVED** [review]

File: locales/cs.lua [hunk 1]
```diff
@@ -26,3 +39,11 @@ if GetConvar('qb_locale', 'en') == 'cs' then
...
     })
 end
+if GetConvar('qb_locale', 'en') == 'cs' then
+    Lang = Locale:new({
+        phrases = Translations,
+        warnOnMissing = true,
+        fallbackLang = Lang,
+    })
+end
+--translate by stepan_valic
```
Comment (solareon) [inline locales/cs.lua:49]: Please remove this comment. The git commit has the author name attached to it.

Comment (stepanvalic) [reply locales/cs.lua:42]: fixed

Comment (MafewTM) [reply locales/cs.lua:42]: It's not fixed. Your comment is still there.

Comment (stepanvalic) [reply locales/cs.lua:42]: I'll try it tomorrow

**Review by Manason — DISMISSED** [review]
Resolve conflicts

**Review by mafewtm — APPROVED** [review]

**Pull request merged**